tests/jobs/overburden_timeshift/test_ots_config.py

```diff
@@ -0,0 +1,68 @@
+import yaml
+import pytest
+from semeio._exceptions.exceptions import ConfigurationError
+from semeio.jobs.overburden_timeshift.ots import ots_load_params
+from semeio.jobs.overburden_timeshift import ots
+from unittest import mock
+from datetime import datetime
+
+
+def test_valid_config(tmpdir, monkeypatch):
+    dates = ["1997-11-06", "1997-12-17", "1998-02-01", "1998-02-01"]
+    context_mock = mock.Mock(
+        return_value=[datetime.strptime(x, "%Y-%m-%d").date() for x in dates]
+    )
+    monkeypatch.setattr(ots, "extract_ots_context", context_mock)
+    conf = {
+        "eclbase": "NORNE_ATW2013",
+        "above": 100,
+        "seabed": 300,
+        "youngs": 0.5,
+        "poisson": 0.3,
+        "rfactor": 20,
+        "mapaxes": False,
+        "convention": 1,
+        "output_dir": "ts",
+        "horizon": "horizon.irap",
+        "ascii": "ts.txt",
+        "velocity_model": "norne_vol.segy",
+        "vintages": {
+            "ts_simple": [["1997-11-06", "1998-02-01"], ["1997-12-17", "1998-02-01"]],
+            "dpv": [["1997-11-06", "1997-12-17"]],
+        },
+    }
+    with tmpdir.as_cwd():
+        with open("ots_config.yml", "w") as f:
+            yaml.dump(conf, f, default_flow_style=False)
+        ots_load_params("ots_config.yml")
+
+
+def test_invalid_config(tmpdir, monkeypatch):
+    dates = []
```

Comment:
context_mock can go away and it should trigger the ConfigurationError anyway

Reply:
It also triggers `util_abort`

Reply:
Then there might be another issue? Why is util abort triggered? (He asked, well knowing the answer can be awkward ;) )

Reply:
There is another issue, related to reading an ecl file which isnt ecl I think. My plan is to create an issue once this is merged (so I can reference the code). The configuration should in general have better test coverage.

Reply:
That shouldn't throw a util abort, but a more general error which can be caught (although that error is thrown for a lot of things..). Util abort is rather thrown if the ecl file is corrupt, and ecl can't recover.. but for later I guess